structure/0006-governance.md

```diff
@@ -2,53 +2,161 @@
+
+- **Motivation**:
+  - Why is this team needed?
+  - What problems will this team solve?
+  - What tasks will this team be responsible for?
+
+- **Explanation**:
+  - What will this team *do*?
+  - What new projects will be created that this team will be responsible for?
+  - How will each of these projects benefit the Quilt organization/community?
+  - What processes will this team follow for each of the different kinds of changes their projects may undergo?
+
+- **Drawbacks**:
+  - Is there overlap of responsibilities?
+  - Is there excessive granularity?
+
+- **Rationale and Alternatives**:
+  - Is there already a team responsible for the described tasks?
+  - If this RFC doesn't pass, which team would be responsible for the described tasks?
+
+- **Prior Art**:
+  - Are there other organizations or projects with comparable tasks?
+
+### Sub Teams
+
+Some teams may wish for more granular organization of their structure.
+In those cases, these teams may create sub teams.
+
+Sub teams behave like regular teams, except they are not required to have a team lead.
+If the sub team doesn't have its own team lead, the parent team lead's acts as its team lead.
+In general, a sub team inherits its parent's structure according to their parent's team RFC.
+
+### Triage Teams
+
+A special type of teams called Triage Teams can exist. They are here to provide support to the organization, through tasks
+such as triaging and tagging GitHub issues, quality assurance, writing regression tests, etc..
 
-The vote must be initiated with a detailed call to action, describing why the staff member feels that the person should be removed from power. The accused will have one week to submit a detailed counterargument, at which point the vote will be called.
+Members of Triage Teams aren't considered organisation members.
 
-### The Administrative Board
-The administrative board is the group of staff that holds ultimate power over the direction of the project. It is made up of three or more members so as to prevent deadlocks in the decision-making process. Members of the administrative board may be members of other teams within the project as well. Admins do not generally make use of their position unless a critical decision needs to be made whose team is unable to come to a consensus. In this case, the board will debate internally until they come to a decision. The administrative board acts primarily as a tiebreaker.
+## The Administrative Board
 
-If there are less than three members on the administrative board, each remaining board member may nominate a community member (potentially a non-staff member) for the position. Existing staff members may nominate themselves. The remaining staff members may then vote on the nominees, with administrative board member votes being worth three votes only in the event of a tie.
+The administrative board is a group of organization members responsible for tasks that are not assigned to another team.
```

Comment:
Is the administrative board considered a team, as defined above?

Reply:
No, they don't have any team lead